doc/en/components/grids/_shared/column-hiding.md

```diff
@@ -744,7 +948,21 @@ In order to achieve that, we will use a class that we will first assign to the g
 ```tsx
 <{ComponentSelector} className="grid"></{ComponentSelector}>
 ```
-<!-- ComponentEnd: Grid, TreeGrid -->
+<!-- ComponentEnd: Grid -->
+
+<!-- ComponentStart: TreeGrid -->
```

Comment:
The snippets from this section should have id and not class/className to be aligned with the [styling snippet](https://github.com/IgniteUI/igniteui-xplat-docs/pull/1265/files#diff-eaf5cb834be03f319ad76a652aedb248e7d9f5837cac6e82649738283688af71R1018-R1048) and the generated [sample](https://github.com/IgniteUI/igniteui-xplat-examples/pull/585/files#diff-116e9176fff13e907f20187fcadee8c62f34ca8de10b5c465cad2b2cc853a904).